hpfeeds-logger/hpfeedslogger/processors.py

```diff
@@ -111,7 +120,7 @@ def glastopf_event(identifier, payload):
     try:
         dec = ezdict(json.loads(str(payload)))
     except:
-        print 'exception processing glastopf event'
+        logger.warning('exception processing glastopf event')
         traceback.print_exc()
```

Comment:
in hpfeeds-cif you put this traceback into the logger stream; same treatment here?

Reply:
Several more of these later in the file

Reply:
Honestly, I put the traceback in the logger stream so I could see it while testing (it was throwing an error, but not actually displaying the error message). Honestly, we should probably just assign the error to a variable in these, and log those errors.